agents/agents-features/agents-features-opentelemetry/src/jvmMain/kotlin/ai/koog/agents/features/opentelemetry/attribute/attributes.kt

```diff
@@ -18,7 +18,7 @@ internal fun List<Attribute>.toAttributes() : Attributes {
         override fun forEach(consumer: BiConsumer<in AttributeKey<*>, in Any>) {
             forEach { attribute ->
                 val attributeKey = AttributeKey.stringKey(attribute.key)
-                consumer.accept(attributeKey, attribute.value)
+                consumer.accept(attributeKey, attribute.value.toString())
```

Comment:
Should be fixed in the upcoming commit with fixers for OT support. Please see https://github.com/JetBrains/koog/pull/401